libp2p/connmanager.nim

```diff
@@ -169,7 +170,7 @@ proc triggerPeerEvents*(c: ConnManager, peerId: PeerId, event: PeerEvent) {.asyn
   except CancelledError as exc:
     raise exc
   except CatchableError as exc: # handlers should not raise!
-    warn "Exception in triggerPeerEvents", exc = exc.msg, peer = peerId
+    warn "Exception in triggerPeerEvents", errMsg = exc.msg, peer = peerId
```

Comment:
What about `errorMsg`?

Reply:
I decided to go with that one because `errMsg` is already widely used across the codebase.

Reply:
I see, I didn't realize it. Thanks